incubator/java-openliberty/image/project/pom.xml

```diff
@@ -54,31 +54,43 @@
         </pluginManagement>
         <plugins>
             <!-- Enforcing OpenLiberty and JDK Version -->
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-enforcer-plugin</artifactId>
-                <version>3.0.0-M2</version>
-                <executions>
-                <execution>
-                    <id>enforce-versions</id>
-                    <goals>
-                        <goal>enforce</goal>
-                    </goals>
-                    <configuration>
-                        <rules>
-                            <requireJavaVersion>
-                                <version>[1.8,1.9)</version>
-                            </requireJavaVersion>
-                            <requireProperty>
-                                <property>version.openliberty-runtime</property>
-                                <regex>{{.stack.libertyVersion}}</regex>
-                                <regexMessage>OpenLiberty runtime version must be {{.stack.libertyVersion}}</regexMessage>
-                            </requireProperty>
-                        </rules>
-                    </configuration>
-                </execution>
-            </executions>
-            </plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-enforcer-plugin</artifactId>
+				<version>3.0.0-M3</version>
+				<executions>
+					<execution>
+						<id>default-cli</id>
+						<goals>
+							<goal>enforce</goal>
+						</goals>
+						<configuration>
+							<rules>
```

Comment:
Some thoughts:
1. We should err on side of less restriction.  Let Champ be more restrictive if he wants and let this be a bit of an example as to how, but unless we're helping our direct stack users somehow we should do less rather than more.
2. We could just check/enforce the plugin version property but allow Jane to configure the whole plugin if she's motivated enough to do so?      The fact that the config is complicated isn't that bad .. but maybe since I'm really on the fence it's swaying me over to not worrying about the plugin (but just the property).   
3. Maybe it'd help to ban the wasdev plugin v2.x?  In case someone copied/pasted some config from the internet?
4.  enforcing the OL version is good , since it makes it easier to align with the `appsody build`.

Reply:
For this particular move (v3.1 -> v3.2), I think we want to be more restrictive since we are modifying our stack to no longer work properly with v3.1. (watching for changes in the pom.xml). Typically, id assume that if we upgraded our stack to use a newer LMP version, it would be to fix a potential bug or add some additional enhancement. I agree it would be good to allow Jane to use any version _above_ what is defined in the stack, but not sure if the enforcer plugin can be configured that way.